### PR TITLE
Rps 5874 update our public python sdk to a tier 2 basic api class reworked

### PR DESCRIPTION
**Pull request by anatolija**

### CODE REVIEW

File: smartlingApiSdk/ApiV1.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+#FileApi V1 class implementation
+
+from .HttpClient import HttpClient
+from .MultipartPostHandler import MultipartPostHandler
+from .Constants import Uri, Params, ReqMethod
+from .ApiResponse import ApiResponse
+from .FileApiBase import FileApiBase
+
+class ApiV1(FileApiBase):
```
Comment (junky) [inline smartlingApiSdk/ApiV1.py:25]: Do we really need to support V1?

**Review by junky — APPROVED** [review]

File: smartlingApiSdk/ApiV1.py (hunk 1 above)
Comment (anatolija) [reply smartlingApiSdk/ApiV1.py:25]: re-checked with fileAPI team, we had  11,076 v1 api calls for last 24hours that are not my tests:
`https://slc.smartling.net/en-GB/app/search/search?q=search%20env%3D%22apigateway%22%20service_id%3DTMS-PUBLIC-API%20%22%2Fv1%2Ffile%2Fget%22%20user_agent%3D*python*%20%20NOT%20%22v2.2.3%22&display.page.search.mode=smart&dispatch.sample_ratio=1&earliest=-24h%40h&latest=now&sid=1627029528.33168910`

Comment (anatolija) [reply smartlingApiSdk/ApiV1.py:25]: What if we remove examples for v1 so new user will check v2 examples only ?

Comment (junky) [reply smartlingApiSdk/ApiV1.py:25]: good idea

Comment (anatolija) [reply smartlingApiSdk/ApiV1.py:25]: I'll also rename tests and v1 class to mark it's obsolete

**Review by junky — APPROVED** [review]

**Pull request merged**